DOC-2023 Troubleshooting steps for NTP issue on Palette/VerteX Management Appliance Installations (#7338)
spectrocloud/librarium: docs/docs-content/troubleshooting/enterprise-install.md

Refer to the following sections to troubleshoot errors encountered when installing an Enterprise Cluster.

## Scenario - Unexpected Logouts in Tenant Console After Palette/VerteX Management Appliance Installation

After installing self-hosted Palette/Palette VerteX using the
[Palette Management Appliance](../enterprise-version/install-palette/palette-management-appliance.md) or
[VerteX Management Appliance](../vertex/install-palette-vertex/vertex-management-appliance.md), you may experience
unexpected logouts when using the tenant console. This can be caused by a time skew on your Palette/VerteX management
cluster nodes, which leads to authentication issues.

> To check this, open a terminal session on each node in your Palette/VerteX management cluster and issue the following

Reviewer: ```suggestion
To validate this setting, open a terminal session on each node in your Palette/VerteX management cluster and issue the following
```

Author: ```suggestion
To verify the system time, open a terminal session on each node in your Palette/VerteX management cluster and issue the following
```

Refer to the following sections to troubleshoot errors encountered when installing an Enterprise Cluster.

## Scenario - Unexpected Logouts in Tenant Console After Palette/VerteX Management Appliance Installation

After installing self-hosted Palette/Palette VerteX using the
[Palette Management Appliance](../enterprise-version/install-palette/palette-management-appliance.md) or
[VerteX Management Appliance](../vertex/install-palette-vertex/vertex-management-appliance.md), you may experience
unexpected logouts when using the tenant console. This can be caused by a time skew on your Palette/VerteX management
cluster nodes, which leads to authentication issues.

To check this, open a terminal session on each node in your Palette/VerteX management cluster and issue the following
command to check the system time.

```bash
timedatectl
```

> You will see output similar to the following. A time skew is indicated by the `Local time` and `Universal time` values

Author: ```suggestion
An output similar to the following will be displayed. A time skew is indicated by the `Local time` and `Universal time` values
```

y' got authorized] [orgId= sessionId=ally5a03 ]
auth-5f95c77cb-xslmk Jul  7 17:22:12.673 ERROR [hubble_token.go:426/hucontext.getClaimsFromToken] [Unable to parse the token 'abcd...1234' due to Token used before
issued]
auth-5f95c77cb-xslmk Jul  7 17:22:16.004 INFO [auth_login.go:450/internal.(*AuthLoginService).CreateUserToken] [reqId:r1751908935214posa171c4b915ad] [Auth success:
12345abc67890def12345abc,johndoe/johndoe@company.com - Going for MFA false] [orgId=johndoe tenantUid=12345abc67890def12345abc ]
auth-5f95c77cb-xslmk Jul  7 17:22:46.378 ERROR [hubble_token.go:426/hucontext.getClaimsFromToken] [Unable to parse the token 'abcd...1234' due to Token used before
issued]
auth-5f95c77cb-xslmk Jul  7 17:22:46.378 ERROR [auth_service.go:282/service.(*AuthService).Logout] [provided token 'xxxxx' is not valid Token used before issued]
auth-5f95c77cb-xslmk Jul  7 17:22:57.473 INFO [auth_login.go:450/internal.(*AuthLoginService).CreateUserToken] [reqId:r1751908976769pos6533ca18ccb3] [Auth success:
12345abc67890def12345abc,johndoe/johndoe@company.com - Going for MFA false] [orgId=johndoe tenantUid=12345abc67890def12345abc ]

> ``` -->

Author: Hidden this for now as I'm awaiting the exact log file where QA provided this example.

It would be a nice enhancement, although it's not entirely needed.

Author: I got the details from QA and have now included this part.
